networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,88 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-4"]
+== Network Observability Operator 1.4.0
+The following advisory is available for the Network Observability Operator 1.4.0:
+
+* link:https://errata.devel.redhat.com/advisory/2023:116729[RHSA-2023:1817 Network Observability Operator 1.4.0]
+
+[id="network-observability-channel-removal-1.4"]
+=== Channel removal
+You must switch your channel from `v1.0.x` to `stable` to receive the latest Operator updates. The `v1.0.x` channel is now removed.
+
+[id="network-observability-operator-1.4.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-without-loki-1.4"]
+==== Network Observability without Loki
+The Network Observability Operator is now functional and usable without Loki. If Loki is not installed, it can only export flows to KAFKA or IPFIX format and provide metrics in the Network Observability metrics dashboards. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki]. 
+
+[id="network-observability-enhanced-configuration-and-ui-1.4"]
+==== Notable enhancements
```

Comment:
This may not be something you want to change, but I was thinking maybe you want to put the notable enhancements before the general new features and enhancements. You could adjust the headings accordingly if you wanted to do that too. So,

> [id="network-observability-operator-1.4.0-features-enhancements"]
> === New features and enhancements
> [id="network-observability-enhanced-configuration-and-ui-1.4"]
> ==== Notable enhancements
> [id="network-observability-without-loki-1.4"]
> ===== Network Observability without Loki

I think what I am trying to get at is the entry "Network Observability without Loki" seems out of place where it is.

Reply:
Yeah I can totally see that now that you point it out. I like the reorg, suggestion taken :)